knimin/lib/data_access.py

```diff
@@ -3191,6 +3186,252 @@ def delete_sample_plate(self, id):
             TRN.add(sql, [id])
             TRN.execute()
 
+    def get_id_by_name(self, field, name):
```

Comment:
@josenavas Yes you did. Sorry! pm_plate_list itself is very simple and contains no JavaScript at all. The Python code in data_access.py is the real complicated one. Btw, thanks for answering #4. After that I edited the Python codes substantially. Then I did the unit test for the additional Python functions.